src/components/InlineFilter/index.tsx

```diff
@@ -0,0 +1,34 @@
+import type React from 'react';
+import FilterChildren from '../FilterChildren';
+
+type InlineFilterProps = {
+  children: JSX.Element
+  filters?: string[]
+  filter?: string
+}
+
+export default function InlineFilter({ filter, filters, children }: InlineFilterProps) {
+  if (filter && filters) {
+    throw new Error("Can't set both `filter` and `filters` props.")
```

Comment:
Do we want to throw here? If someone does this by mistake and we don't catch it in PR, won't this whitescreen the page? cc @katiegoines